### DISCUSSION TITLE
0.7.0.beta.1 for 1.20.1 and 1.21

### DISCUSSION THREAD

**Discussion by omo50** [Nostalgica-Reverie/Content-Monorepo#105] [General]
# Re-Console 0.7.0.beta.1 for 1.20.1 and 1.21
Here's the first beta for 0.7.0. Comes with a backport to 1.20.1, alongside additions for user QoL, more OptiFine parity, and more

# Highlights

### Added Animatica, Entity Sound Features and CIT Resewn (CIT on 1.20.1 only)
Implementing more OptiFine resource pack support!

### Added CalcMod
CalcMod is a simple, lightweight and easy to use calculator in chat with extra things designed specifically for Minecraft! It can do anything from basic calculations to nether portal calculations, farm rate calculations and more.

### Added Iris Shaders
Shaders have been a widely requested feature for quite a while now, and I don't think it hurts to add. Completely optional! Access the menu with O

### Added Sodium Extra and Reese's Sodium Options
Due to the ability to access the Sodium options menu via the Mods screen, I believe these mods give the user more control over what they can change while looking nicer in the process!

### Added Old Combat Mod
This simply re-adds the old combat that was used in the pack previously, replacing CombatEdit

### Backported to 1.20.1
Another highly requested thing is 1.20.1 support, due to its larger array of mods than 1.20.4, 1.20.6 and 1.21 as of right now. This makes Re-Console available on 1.20.1 whilst also keeping 1.21 support. This version lacks Nvidium due to it having a memory leak on linux systems.

### Removed FastQuit
FastQuit has shown itself to be problematic in terms of issues regarding world saving, causing chunk errors and data loss, which is why it is no longer present

### Removed Forge Config API Port
A lib that hasn't been used for a bit now since Golden Age Combat was removed in 0.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ViolaFlower/Re-Console/releases/tag/0.7.0.beta.1'>0.7.0.beta.1 for 1.20.1 and 1.21</a>.</em>

**Comment by Technocality** [comment]
Done
